docs/guides/5_setup_aligned.md

```diff
@@ -322,205 +335,169 @@ ecdsa:
   private_key_store_password: <ecdsa_private_key_store_password>
 ```
```

Comment:
Batcher config seems to be old, it's missing:
```
max_proof_size
max_batch_size
eth_ws_reconnects
pre_verification_is_enabled
 ```

Reply:
Solved in [44a26ab](https://github.com/yetanotherco/aligned_layer/pull/631/commits/44a26abfc3b2e462c51f305397b5532d3ee122e2)